src/megatron/bridge/utils/safe_pickle.py

```
@@ -0,0 +1,60 @@
+            "str",
+            "int",
+            "float",
+            "bool",
+            "complex",
+            "slice",
+            "range",
+            "type",
+            "NoneType",
+        },
+        "collections": {"OrderedDict"},
+    }
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)